README.md

```diff
@@ -40,7 +40,7 @@ Please join us during our online community meetings. Details can be found on our
 
 You can chat with us on Kubernetes Slack in the #carvel channel and follow us on Twitter at @carvel_dev.
 
-Check out which organizations are using and contributing to Carvel: [Adopter's list](https://github.com/vmware-tanzu/carvel/blob/master/ADOPTERS.md)
+Check out which organizations are using and contributing to Carvel: [Adopter's list](https://github.com/carvel-dev/carvel/blob/master/ADOPTERS.md)
```

Comment:
should we also change the branch from master to develop here? (not a blocker as it still works today as it redirects to the default branch with popup)
```suggestion
Check out which organizations are using and contributing to Carvel: [Adopter's list](https://github.com/carvel-dev/carvel/blob/develop/ADOPTERS.md)
```

Reply:
Sure, but maybe a separate PR?

Reply:
Yeah.